server/meta.jsx

```diff
@@ -360,6 +360,21 @@ export function unpackSDKMeta(sdkMeta?: string): SDKMeta {
                                 version = '';
                             }
 
+                            // if patch version is less than 132, we want to 
+                            // set the version to 132. If for some reason we can't
+                            // parse out a patch version, set version as latest
+                            // if neither cases are true, leave version alone because
+                            // it can be more than a semver version number ("min" for example)
+                            if (/4\\.0\\.\\d{1,3}/.test(version)) {
+                              var patchString = version?.split('.')?.pop()
+
+                              if (!patchString) {
+                                version = 'latest'
```

Comment:
above on L359, if we see `latest` then we set `version` to an empty string. Should this be an empty string here instead?

Reply:
oh damn i had that backwards, thank you.

I need to add a test case for if patch version is undefined, i think i know how

Reply:
okay, don't think its possible to have an invalid patch version but i'm leaving that there just in case